### PR TITLE
Purgecss Processor

### PR DESCRIPTION
**Pull request by bemky**
Add PurgeCSS as a post_processor

### CODE REVIEW

**Comment by malomalo** [comment]
Do you want to setup [autolaod](https://github.com/malomalo/condenser/blob/master/lib/condenser.rb#L33)?

File: lib/condenser/processors/node_processor.rb [hunk 1]
```diff
@@ -8,6 +8,10 @@ class NodeProcessor
     
     def self.setup(environment)
     end
+  
+    def name
+      self.class.name
+    end
```
Comment (malomalo) [inline lib/condenser/processors/node_processor.rb:14]: Awesome, can you remove this function from the other processors that inherit from `NodeProcessor` and duplicate this method, then this is good to merge

**Review by malomalo — CHANGES_REQUESTED** [review]

File: lib/condenser/processors/node_processor.rb (hunk 1 above)
Comment (bemky) [reply lib/condenser/processors/node_processor.rb:14]: @malomalo there are not anymore uses of `def name` on the processors

**Comment by malomalo** [comment]
Ah kk, Did you want any test with this?

**Review by malomalo — APPROVED** [review]

**Pull request merged**